pkg/download/id_extraction/id_extraction.go

```diff
@@ -31,36 +30,42 @@ var meIDRegexPattern = regexp.MustCompile(`[a-zA-Z_]+-[A-Za-z0-9]{16}`)
 
 var uuidRegexPattern = regexp.MustCompile(`[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`)
 
+const baseParamID = "extractedIDs"
```

Comment:
I am also not fully happy, but it describes IMO best what it is.
Extracted could be too broad just in case we add something else in the future in a different sub-field (e.g. emails)